src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatWarning.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Sonar Scala Plugin
+ * Copyright (C) 2018 All contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package com.mwz.sonar.scala.scapegoat
+
+/** Represents a warning in the scapegoat report */
+private[scapegoat] final case class ScapegoatWarning(
```

Comment:
I realise that coincidentally Scapegoat calls this a `warning` in the report, but maybe we could rename it to `ScapegoatIssue` so the name doesn't clash with Scapegoat 'issue' severity level - Warning? 😄

Reply:
Yeah, it is really confusing.

Reply:
I'm sorry, I already change the name of the class, but I didn't change the name of the variables with instances of this class. I'll fix it in a couple of hours